README.md

```diff
@@ -235,15 +235,21 @@ Amber is a community effort and we want You to be part of it. [Join Amber Commun
 4. Push to the branch (git push origin my-new-feature)
 5. Create a new Pull Request
 
-## Amber Core Team
+## Amber Team
```

Comment:
**Amber Maintainers**
- Dru Jensen [drujensen](https://github.com/drujensen) 
- Elias Perez [eliasjpr](https://github.com/eliasjpr) 
- Isaac Sloan [elorest](https://github.com/elorest) 
- Faustino Aguilar [faustinoaq](https://github.com/faustinoaq) 
- Nick Franken [fridgerator](https://github.com/fridgerator)
- Mark Siemers [marksiemers](https://github.com/marksiemers)
- Robert Carpenter [robacarp](https://github.com/robacarp) 

Same for contributors